src/internal/internal_util.hh

```diff
@@ -109,6 +109,21 @@ slate::Matrix<scalar_t> alloc_basis(slate::BaseMatrix<scalar_t>& A, int64_t n,
     return V;
 }
 
```

Comment:
NB: it has been suggested that SLATE matrices store offsets instead of tileNb, in which case tileMb( i ) = tileOffset( i+1 ) – tileOffset( i ). Future work.